src/eddymotion/math/utils.py

```diff
@@ -43,3 +43,31 @@ def is_positive_definite(matrix):
     except np.linalg.LinAlgError:
         # Matrix is not positive definite
         return False
+
+
+def compute_angle(v1, v2, smallest=False):
+    """Compute the angle between two vectors.
+
+    Parameters
+    ----------
+    v1 : :obj:`ndarray`
+        First vector.
+    v2 : :obj:`ndarray`
+        Second vector.
+    smallest : :obj:`bool`
+        ``True`` to compute the smallest angle between the two vectors.
+
+    Returns
+    -------
+    :obj:`float`
+        Angle between vectors.
+    """
+
+    dot_product = np.dot(v1, v2)
+    norm_v1 = np.linalg.norm(v1)
+    norm_v2 = np.linalg.norm(v2)
+    cosine_angle = dot_product / (norm_v1 * norm_v2)
```

Comment:
Perhaps it is clearer if we use the operator and ensure unit vectors first:

```suggestion
    cosine_angle = (v1 / np.linalg.norm(v1)) @ (v2 / np.linalg.norm(v2))
```

Reply:
Done in 3fce5be.